src/app/models/evt-models.ts

```diff
@@ -105,3 +105,27 @@ export interface NamedEntityRefData extends GenericElementData {
     entityId: string;
     entityType: NamedEntityType;
 }
+
+export interface WitnessesData {
+    witnesses: Map<Witness>;
+    witnessGroups: WitnessGroups;
+}
+
+export interface Witness {
+    id: string;
+    name: Description;
+    attributes: AttributesData;
+    content: Description;
+    group: string;
+}
+
+export interface WitnessGroups {
+    groups: Map<WitnessGroup>;
+}
+export interface WitnessGroup {
+    id: string;
+    name: string;
+    attributes: AttributesData;
+    content: string[];
```

Comment:
If this is intended to list the ids of witnesses, I would change the property name in something like `witnesses` or `witnessesIds`. If not, ignore my comment.